worldedit_commands/init.lua

```diff
@@ -456,15 +456,35 @@ minetest.register_chatcommand("/replaceinverse", {
 })
 
 local check_sphere = function(name, param)
-	if worldedit.pos1[name] == nil then
-		worldedit.player_notify(name, "no position 1 selected")
-		return nil
-	end

```

Comment:
Thanks! This was removed by accident when removing the guarding conditional that was in place before implementing sphere support.